cli/cmd/main.go

```diff
@@ -303,6 +305,16 @@ func main() {
 								Required: false,
 								Usage:    "ISO-8601 timestamp indicating the end of the interval filter",
 							},
+							&cli.StringFlag{
```

Comment:
I feel like it's a little weird for this to be split across two commands, since both are for the end user use case of "give me data with tags <anything/nothing/specific-things>". 

I think it might make sense for them to just be one command and "tagged" and "untagged" to basically be keywords to the `dataFlagTags` command, i.e. if they pass `tags=tagged` it would be equivalent to to `dataFlagTaggedUntagged=tagged` currently. Open to other opinions on how best to combine them, but I do think they should probably be combined

Reply:
I think that's a reasonable solution, agree that want to keep it simple in the CLI versus the explicit nested tags/types that we expose via the API.

So they can pass `tags=tagged`, `tags=untagged`, or `tags=square,triangle,circle`. Would explain that in the usage text, e.g. "indicates filtering based on data with tags. accepts `tagged` for all tagged data, `untagged` for all untagged data, or a list of tags for all data matching any of the tags"

If they're specifically only trying to filter on a single tag that's named "tagged" or "untagged" this fails but don't think we need to worry about that :)

Reply:
done!

if they are trying to filter on "tagged" or "untagged",
think that `tags=,tagged` would do the trick